melos.yaml

```diff
@@ -8,29 +8,31 @@ packages:
 ide:
   intellij:
     enabled: true
-    moduleNamePrefix: ''
-  vscode: true
```

Comment:
`vscode` 
Removed non-existent configuration value.

Reply:
`moduleNamePrefix` should persist, no?

Reply:
OK, I don't have a strong preference regarding this, so I'll leave it as it is.

Reply:
This helps when using the IDEA so those module won't have a redundant `melos` prefix.